test/e2e/specs/templates.test.js

```diff
@@ -60,12 +62,12 @@ describe( 'templates', () => {
 		const STANDARD_FORMAT_VALUE = '0';
 
 		async function setPostFormat( format ) {
+			await switchToAdminUser();
 			await visitAdmin( 'options-writing.php' );
 			await page.select( '#default_post_format', format );
-			return Promise.all( [
```

Comment:
This was accounting for a race condition acknowledged in the Puppeteer docs:

>Bear in mind that if click() triggers a navigation event and there's a separate page.waitForNavigation() promise to be resolved, you may end up with a race condition that yields unexpected results. The correct pattern for click and wait for navigation is the following:

https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageclickselector-options